jenkins.jq

```diff
@@ -47,3 +49,53 @@ def gha_payload:
 		)
 	}
 ;
+
+# input: full "build" object list (with "buildId" top level key)
+# output: filtered build list { "buildId value": { build object } }
+def get_arch_queue($arch):
+	map_values(
+		select(
+			needs_build
+			and .build.arch == $arch
+		)
+		|  if .build.arch | IN("amd64", "i386", "windows-amd64") then
+			# "GHA" architectures (anything we add a "gha_payload" to will be run on GHA in the queue)
+			.gha_payload = (gha_payload | @json)
+		else . end
+		| .identifier = (.source.arches[.build.arch].tags[0] + if .build.arch != $arch then " (\(.build.arch))]" else "" end)
```

Comment:
On second thought, with the `select` in here, this doesn't even make sense to include anymore :sob: (it will literally never be true and we should think about it when/if we ever bring that behavior back)

```suggestion
		| .identifier = .source.arches[.build.arch].tags[0]
```